vaccine_feed_ingest_schema/location.py

```diff
@@ -103,7 +234,7 @@ class Vaccine(BaseModel):
     }
     """
 
-    vaccine: str
+    vaccine: VaccineType
     supply_level: Optional[str]
```

Comment:
Should we add an enum for this?

Reply:
Yes, I was unsure of the enum values though. I think only one source has this info?

Reply:
Previously we listed [these](https://github.com/CAVaccineInventory/vaccine-feed-ingest/wiki/Normalized-Location-Schema#supply-level-enum), but I agree this has not been broadly used so far.

Should we use that set (or maybe just `in_stock` / `out_of_stock`) as a starting point, and can add to it in future releases?

Reply:
Sure. It looks like `in_stock` is the only value currently being used: https://github.com/CAVaccineInventory/vaccine-feed-ingest/search?q=supply_level

Reply:
At least [one](https://github.com/CAVaccineInventory/vaccine-feed-ingest/pull/226/) of the open PRs use `out_of_stock` also, but in that case it looks like it might be being used to mean "not at this location."

Reply:
Added enum with in_stock / out_of_stock and changed to wiki to reflect those two values